### PR TITLE
step 2 fix

### PR DESCRIPTION
**Pull request by uzfarid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,9 +57,20 @@ VPC and Subnets can be found under the VPC Service.
...
+Create new EC2 instance using ICAPServer-Performance-Analytics-Dashboard - ami-039215eee67c4041e image from AWS community using the following steps:
+
+- Click on Launch Instance
+- Type "icapserver" in search field & click enter
+- Click on "Community AMIs"
+- Select "ICAPServer-Performance-Analytics-Dashboard" AMI
+- Select t3.medium (for average use) instance type
+- Select desired VPC and public subnet (created in step1, or existing own one can be used)
+- Auto Assign Public IP & leave all other options as default
+- Click Add Storage, then Click Add Tags : Create Name tag to identify your instance
```
Comment (jpoolGW) [inline README.md:69]: Should we add a suggested storage size to mount to ec2 instance?

File: README.md [hunk 2]
```diff
@@ -57,9 +57,20 @@ VPC and Subnets can be found under the VPC Service.
...
+- Next Configure security group: Select ICAP-Performance-Dashboard-SG created in step 1
+- Review & Launch. Select your own keypair.
```
Comment (jpoolGW) [inline README.md:71]: Or create one

File: README.md [hunk 3]
```diff
@@ -57,9 +57,20 @@ VPC and Subnets can be found under the VPC Service.
...
+- Open Browser and enter http://[instance public ip]:3000
+- Grafana ui opens and login with username/password: admin/glasswall
```
Comment (jpoolGW) [inline README.md:73]: Should we advise they edit the password after first login?

**Review by jpoolGW — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (uzfarid) [reply README.md:69]: storage size needed comes as default

File: README.md (hunk 3 above)
Comment (uzfarid) [reply README.md:73]: they can follow own security policies to decide edit or not edit

**Review by jpoolGW — APPROVED** [review]

**Pull request merged**